src/lrstar/PG_Generate.cpp

```diff
@@ -2027,6 +2027,7 @@ static void instantiate_main(FILE *fp, const char *grammar)
            "        delete [] input_start;\n"
            "    }\n\n"
            "    fclose(output_fp);\n"
+           "    if(options.output) free(options.output);\n"
```

Comment:
Please add a new block for the free().

if (options.output) {
  free(options.output);
}

Or simply free(options.output), because free-ing a NULL pointer is ok.